lambdas/service/app.py

```diff
@@ -179,37 +233,55 @@ def openapi():
 
 
 health_spec = {
```

Comment:
As long as the responses are the same for all of the `method_spec`s, then it makes sense to use this variable to reduce duplication.